### DISCUSSION TITLE
🚀 Role - Custom URI - URL cleanup function like in Menu Manager

### DISCUSSION THREAD

**Discussion by ic360-com** [ExpressionEngine/ExpressionEngine#4215] [Feature Requests]
### Suggest an idea for ExpressionEngine

When pasting in a URL from within the CP... would it be possible to have same URL cleanup function, as used in the Menu Manager?

**Comment by intoeetive** [comment]
@ic360-com exactly where are you pasting the URL and need the cleanup? (which pages/fields)

**Comment by ic360-com** [comment]
It's on the Edit Role screen - the Customer URI field.

![Screenshot 2024-04-12 at 11 10 00](https://github.com/ExpressionEngine/ExpressionEngine/assets/36453035/d130b9dd-1c53-4b1f-beff-527a97955069)
